src/components/ObsDetails/ObsDetails.js

```diff
@@ -39,24 +38,26 @@ const ObsDetails = ( ): Node => {
       <ScrollView>
       <View style={viewStyles.userProfileRow}>
         <Pressable style={viewStyles.userProfileRow} onPress={navToUserProfile}>
-          <UserIcon uri={observation.userProfilePhoto} />
-          <Text>{`@${observation.userLogin}`}</Text>
+          {/* TODO: fill user icon in with saved current user icon or icon from another user API call */}
+          <UserIcon uri={null} />
+          {/* TODO: fill in text with saved current user login or login from another user API call */}
+          <Text>@username</Text>
```

Comment:
That's a good idea -- I can make a line item for that in the mobile dev timeline. I've been curious about libraries like [react-native-skeleton-content](https://github.com/alexZajac/react-native-skeleton-content-nonexpo) which generate a skeleton UI, since that seems like the direction most mobile apps have gone instead of loading wheels. But that seemed to me like a design decision and also wouldn't solve for the empty state you're taking about.